pkg/agent/route/route_linux.go

```diff
@@ -1614,45 +1844,36 @@ func (c *Client) addVirtualServiceIPRoute(isIPv6 bool) error {
 	return nil
 }
 
-// AddNodePort is used to add IP,port:protocol entries to target ip set when a NodePort Service is added. An entry is added
-// for every NodePort IP.
-func (c *Client) AddNodePort(nodePortAddresses []net.IP, port uint16, protocol binding.Protocol) error {
+// AddNodePortConfigs is used to add IP,protocol:port entries to target ipset when a NodePort Service is added. An
+// entry is added for every NodePort IP.
+func (c *Client) AddNodePortConfigs(nodeIPs []net.IP, svcNodePort uint16, protocol binding.Protocol) error {
 	isIPv6 := isIPv6Protocol(protocol)
-	transProtocol := getTransProtocolStr(protocol)
-	ipSetName := getNodePortIPSetName(isIPv6)
-
-	for i := range nodePortAddresses {
-		ipSetEntry := fmt.Sprintf("%s,%s:%d", nodePortAddresses[i], transProtocol, port)
-		if err := c.ipset.AddEntry(ipSetName, ipSetEntry); err != nil {
+	ipsetName := getNodePortIPSetName(isIPv6)
+	protocolStr := toProtocolStr(protocol)
```

Comment:
I would avoid such unrelated naming and order changes:
getTransProtocolStr => toProtocolStr
transProtocol => ipSetName
nodePortAddresses => nodeIPs

They don't look obviously better but caused quite some line changes. And if I remember correctly, the old names were added by you too. 

Please minimize the patch to necessary code needed to implement the feature, and use separate patches to improve code style of unrelated code if necessary. Otherwise it could be distracting and debatable to discuss this changes, especially for a patch of one thousand lines.

Reply:
Will revert these.